src/xrCore/Animation/BoneEditor.cpp

```diff
@@ -161,26 +136,6 @@ void CBone::BoneRotate(const Fvector& _axis, float angle, bool parentCS /*= fals
             mot_rotate.z += _axis.z * angle;
 
             ClampByLimits();
-            /*
-                        Fmatrix mBind,mBindI,mLocal,mRotate,mLocalBP;
-                        mBind.setXYZi		(rest_rotate);
-                        mBindI.invert		(mBind);
-                        mLocal.setXYZi		(mot_rotate);
-                        Fvector axis;
-                        mBind.transform		(axis,_axis);
-                        mRotate.rotation	(axis,angle);
-                        mLocal.mulA			(mRotate);
-
-                        mLocalBP.mul		(mBindI,mLocal);
-                        Fvector mot;
-                        mLocalBP.getXYZi	(mot);
-
-                        IK_data.clamp_by_limits(mot);
-
-                        mLocalBP.setXYZi	(mot);
-                        mLocal.mul			(mBind,mLocalBP);
-                        mLocal.getXYZi		(mot_rotate);
-            */
```

Comment:
Return deleted code **with** the original formatting.

Reply:
Restored.